couchbase/collection.hxx

```diff
@@ -348,6 +348,13 @@ public:
    * @param options the custom options
    * @return the future object with first available result, might be the active or a replica.
    *
+   * Select preferred server group in connection options:
```

Comment:
Should this section also be added for the callback-based overload of `get_any_replica` above?

Reply:
We do not have ready example for that case yet.